intelmq/lib/bot.py

```diff
@@ -224,53 +235,52 @@ def dump_message(self, ex):
         Load Configurations
     '''
 
-    def load_system_configuration(self):
+    def load_system_configuration(self, config=None):
+        # TODO: Move to defaults?
         setattr(self.parameters, 'logging_path', DEFAULT_LOGGING_PATH)
         setattr(self.parameters, 'logging_level', DEFAULT_LOGGING_LEVEL)
 
-        config = utils.load_configuration(SYSTEM_CONF_FILE)
-        for option, value in config.iteritems():
+        config = config or utils.load_configuration(SYSTEM_CONF_FILE)
+        for option, value in config.items():
             setattr(self.parameters, option, value)
 
-    def load_defaults_configuration(self):
-
+    def load_defaults_configuration(self, config=None):
         # Load defaults configuration section
 
-        config = utils.load_configuration(DEFAULTS_CONF_FILE)
-
-        self.logger.debug("Defaults configuration"
-                          " from '%s' file" % DEFAULTS_CONF_FILE)
+        config = config or utils.load_configuration(DEFAULTS_CONF_FILE)
 
-        for option, value in config.iteritems():
+        for option, value in config.items():
             setattr(self.parameters, option, value)
-            self.logger.debug("Defaults configuration: parameter '%s' "
-                              "loaded with value '%s'" % (option, value))
+            # TODO: Log that really?
 
-    def load_runtime_configuration(self):
+    def load_runtime_configuration(self, config=None):
 
         # Load bot runtime configuration section
 
-        config = utils.load_configuration(RUNTIME_CONF_FILE)
+        config = config or utils.load_configuration(RUNTIME_CONF_FILE)
 
         self.logger.debug("Runtime configuration: loading '%s' section from"
                           " '%s' file" % (self.bot_id, RUNTIME_CONF_FILE))
-
-        if self.bot_id in config.keys():
-            for option, value in config[self.bot_id].iteritems():
+        self.logger.debug("{}".format(list(config.keys())))
+        if '__default__' in list(config.keys()):
```

Comment:
from line 265 to 270 we can remove that....that feature is deprecated... not we have defaults.conf file.